pkg/wallet/api.go

```diff
@@ -20,23 +20,14 @@ type Wallet interface {
 // Crypto interface
 type Crypto interface {
 
-	// CreateEncryptionKey create a new public/private encryption keypair.
+	// CreateKeyPairsCombo create a new public/private encryption and signagure keypairs combo.
```

Comment:
To expand on my "Combo" comment. I would treat this situation more similarly to gpg.

You are simply creating a (messaging) key. Like gpg:
- The encryption subkey happens to be created at the same time. 
- Subkey is associated to a primary key.
- This method should return an identifier for the primary key. This identifier should be derived from the public key.
- You should be able to lookup keys by key identifier.

Reply:
sounds good, then I will restructure this to be a messaging key that represents the signing key and having the encryption key-pair as a subkey.

key identifier will be the public signing key as it will be referenced in DID docs and DIDExchange/Invitation messages as well.

Reply:
And it's important that the identifier be derived deterministically from the (signing) public key, since it's the (signing) public keys that will be packed into an envelope header, which the recipient will need to look up in the KMS metadata to find a usable decryption key.

Reply:
created #596 to deal with the new structure and creating a deterministic id for the keys